option/option.go

```diff
@@ -324,3 +324,15 @@ func (i impersonateServiceAccount) Apply(o *internal.DialSettings) {
 	o.ImpersonationConfig.Delegates = make([]string, len(i.delegates))
 	copy(o.ImpersonationConfig.Delegates, i.delegates)
 }
+
+// WithAttemptDirectPath returns a ClientOption that overrides the default 
+// attempt to use DirectPath
+func WithAttemptDirectPath(dp bool) ClientOption {
```

Comment:
Do we expect users of this library to set this any time soon? Or is this just to be used internally for now? If internal I recommend that we move this to the `internaloption` package.

Reply:
I think a signature like `EnableDirectPath(enable bool)` might be more descriptive. The `With` convention in this package implies that is the thing being passed in as input.

Reply:
Use DirectPath/CFE should be transparent to users, so only Yoshi libraries will need to use this option, so I agree we can move it to `internaloption`. Also I change the name to `EnableDirectPath`.